static/js/src/advantage/react/components/Subscriptions/SubscriptionEdit/SubscriptionEdit.tsx

```diff
@@ -75,7 +76,7 @@ const generateError = (error: Error | null) => {
     return (
       <span data-test="has-pending-purchase-error">
         You already have a pending purchase. Please go to{" "}
-        <a href="/account/payment-methods">payment methods</a> to retry.
+        <a href="/account/payment-methods">payment method</a> to retry.
```

Comment:
we can use `{urls.account.paymentMethods}` instead of `/account/payment-methods` here as well

Reply:
True, I didn't realise

Reply:
True, I didn't realise